src/openrct2/ride/Ride.cpp

```diff
@@ -278,9 +278,9 @@ size_t Ride::GetNumPrices() const
     return result;
 }
 
-int32_t Ride::GetAge() const
+int16_t Ride::GetAge() const
 {
-    return static_cast<int32_t>(gDateMonthsElapsed) - build_date;
+    return static_cast<int16_t>(gDateMonthsElapsed) - build_date;
```

Comment:
Why the size change?

Reply:
Just wanted to keep the return type size consistent with the parameters, but I'm reverting this as per Ted's recommendation.